iothub/device/src/Transport/AmqpIoT/AmqpIoTErrorAdapter.cs

```diff
@@ -25,7 +25,6 @@ internal static class AmqpIoTErrorAdapter
         public static readonly AmqpSymbol IotHubNotFoundError = AmqpIoTConstants.Vendor + ":iot-hub-not-found-error";
         public static readonly AmqpSymbol ArgumentError = AmqpIoTConstants.Vendor + ":argument-error";
         public static readonly AmqpSymbol ArgumentOutOfRangeError = AmqpIoTConstants.Vendor + ":argument-out-of-range";
-        public static readonly AmqpSymbol DeviceAlreadyExists = AmqpIoTConstants.Vendor + ":device-already-exists";
```

Comment:
Service team has confirmed that they never throw this exception to the device client

Reply:
What would happen if you create two clients with same device ID?

Reply:
They would create the ping-pong situation - one client would create a connection -> other client would want to create another connection -> client 1 gets disconnected -> client 2 gets connection -> and so on.